cloudbuild/vendors/testbed.textproto

```diff
@@ -113,6 +113,7 @@ duts {
 duts {
   id: "lemming"
   vendor: OPENCONFIG
+  model: "LEMMING"
```

Comment:
this is not required in the testbed file but is required in the topology textproto, that is what is causing the presub error

Reply:
ACK, apologies. Missed that. Fixed.